dist/constants.js

```diff
@@ -36,9 +36,9 @@ Constants.SDK_VERSION = "2.0.0";
  * @description Maximum log buffer size
  * @static
  * @public
- * @default 12MiB
+ * @default 128MiB
  */
-Constants.MAX_LOG_BUFFER_SIZE = 12582912;
+Constants.MAX_LOG_BUFFER_SIZE = process.env.CORALOGIX_BUFFER_SIZE ? parseInt(process.env.CORALOGIX_BUFFER_SIZE) : 134217728;
```

Comment:
Didn't we've decided not to change they defaults?

Reply:
I rolled this back, it should be outdated change now.